Makefile

```diff
@@ -182,7 +182,7 @@ $(OUT)/emulate.o: CFLAGS += -foptimize-sibling-calls -fomit-frame-pointer -fno-s
 .DEFAULT_GOAL := all
 
 include mk/external.mk
-
+include mk/benchmark.mk
```

Comment:
Shorten as `mk/bench.mk`.

Reply:
For consistency, let's rename `mk/benchmark.mk` to `mk/artifact.mk`, which more accurately reflects its role in generating build artifacts or binary blobs required for [RISC-V Architecture Test](https://github.com/riscv-non-isa/riscv-arch-test) and further evaluation.